webpack.common.js

```diff
@@ -12,11 +12,25 @@ module.exports = {
         filename: '[name].js',
         library: 'opensheetmusicdisplay',
         libraryTarget: 'umd',
-        globalObject: 'this'
+        globalObject: 'this',
+        // Set the publicPath to match the location of the chunk files.
+        publicPath: './', 
+        // TODO: The VexFlow ESM output will need to include the magic comments with webpackChunkName for this to work.
+        chunkFilename: '[name].js' 
     },
     resolve: {
         // Add '.ts' and '.tsx' as a resolvable extension.
-        extensions: ['.ts', '.tsx', '.js']
+        extensions: ['.ts', '.tsx', '.js'],
+        alias: {
+            // This imports the ESM version of VexFlow.
+            // It maps "vexflow" to the "vexflow-core-with-gonville.js" entry file.
+            vexflow: path.resolve(__dirname, "node_modules/vexflow/build/esm/entry/vexflow-core-with-gonville.js")
```

Comment:
This is the magic line.

There's actually a different way to integrate with vexflow-core-with-gonville. I can test it another time.